environment-dev.yml

```diff
@@ -36,6 +36,8 @@ dependencies:
   - scikit-learn
   - pymssql
   - psycopg2
+  - mypy==0.961
```

Comment:
We usually try not to pin. Is there a reason to pin this version? I see we don't pin `black` or `flake8`.

Reply:
I pinned the version to avoid any kind of surprises but I guess there's no reason to do so. I can get rid of the version pinning.